docs/docs/models-definition.md

```diff
@@ -502,7 +502,12 @@ var Bar = sequelize.define('bar', { /* bla */ }, {
   freezeTableName: true,
 
   // define the table's name
-  tableName: 'my_very_custom_table_name'
+  tableName: 'my_very_custom_table_name',
+
+  // Enable optimistic locking.  When enabled, sequelize will add a version count attriubte
+  // to the model and throw an OptimisticLockingError error when stale instances are saved.
+  // Set to true or a string with the attribute name you want to use to enable.
+  version: true
```

Comment:
Is this all documentation? Maybe at least a header "Optimistic locking" that explains what it is, what it does, why it is helpful and how to enable it